ImageConverter/ImageConverter/Input.cs

```diff
@@ -0,0 +1,105 @@
+﻿using ImageConverter.Bmp;
+using ImageConverter.Ppm;
+
+namespace ImageConverter
+{
+    internal class Input
+    {
+        public Input()
+        {
+
+        }
+
+        public void InputCheck(string[] args)
+        {
+            string source = "";
+            string destination = "";
+            string goal_format = "";
+            bool isDestinationParametеrExist = false;
+            bool isSourceParameterExist = false;
+            bool isGoalFormatParameterExist = false;
+
+            if (args.Length > 6)
+            {
+                throw new Exception("Too many parameters");
+            }
+
+            for (int i = 0; i < args.Length; i++)
+            {
+                if (args[i] == "--destination")
+                {
+                    destination = args[i + 1];
+                    isDestinationParametеrExist = true;
+                }
+                if (args[i] == "--source")
+                {
+                    source = args[i + 1];
+                    isSourceParameterExist = true;
+                }
+                if (args[i] == "--goal-format")
+                {
+                    goal_format = args[i + 1];
+                    isGoalFormatParameterExist = true;
+                }
+            }
+
+            if (!isDestinationParametеrExist)
+            {
+                throw new Exception("you forget --destination");
+            }
+            if (!isSourceParameterExist)
+            {
+                throw new Exception("you forget --source");
+            }
+            if (!isGoalFormatParameterExist)
+            {
+                throw new Exception("you forget --goal-format");
+            }
+            if (!File.Exists(source))
+            {
+                throw new Exception("source is not exists");
+            }
+            if (Path.GetExtension(destination) != ".bmp" && Path.GetExtension(destination) != ".ppm")
+            {
+                throw new Exception("destination is wrong");
+            }
+            if (goal_format != ".bmp" && goal_format != ".ppm")
+            {
+                throw new Exception("goal-format must be .ppm or .bmp");
+            }
+            if (Path.GetExtension(destination) == Path.GetExtension(source))
+            {
+                throw new Exception("destination file extension cannot be the same as source file extension");
+            }
+            if (Path.GetExtension(destination) != goal_format)
```

Comment:
should be removed since destination is a folder

Reply:
no distanse is "C:\\Users\\arikt\\Documents\\Programming\\ComputerGraphics\\ImageConverter\\Images\\output23.ppm" where output23 is the name of file and ppm is the file extension